terraform/aks/workspace_variables/sandbox_aks.tfvars.json

```diff
@@ -23,5 +23,6 @@
     "sandbox.api.publish-teacher-training-courses.service.gov.uk"
   ],
   "enable_monitoring": true,
+  "enable_logit": true,
```

Comment:
sandbox is in the prod cluster, so can you remove for this PR as it should be non-prod only.